### ISSUE TITLE
Please consider more visible notifications for save-breaking updates (Request)

### ISSUE THREAD

**Issue by joshuafcole** [zymex22/Project-RimFactory-Revived#332]
**Is your feature request related to a problem? Please describe.**
The most recent update broke my save. That'll have to happen from time to time of course, but it would be nice to know about it in advance. After exploring the discord and the comments, I saw that you _have_ talked about it, but only in forums I don't frequently check.

**Describe the solution you'd like**
It would be stellar if you could add a popup on rimworld startup about save-breaking changes for a week or so before you release them, so people can make the decision about staying on the old version or starting a new save based on that. The proposed flow:
- You decide with the community (as you've done already) to make such a change
- Once the things are good and ready, you communicate a time for release that's at least a week away (or some other duration of your choice -- ideally one that's a reasonable compromise between "get updates into impatient players' hands" and "give folks with less free time a chance to see and deal with the update."
- Then, immediately push a new minor version that _only_ adds the breaking change notification, so it'll be visible for a bit without breaking anything.
- Finally, after that time is up, push the change.

**Describe alternatives you've considered**
It might also be nice to adhere to the [semver spec](https://semver.org/) since your version numbers are formatted like semver. When I saw `2.5.0` I thought it was a non-breaking feature update.

If anyone happens to not be familiar with semver -- in a nutshell version numbers use the format `MAJOR . MINOR . BUGFIX`. Major *must* be incremented on breaking changes, minor *must* be incremented on feature additions, and bugfix is incremented for bugfixes only (so this version would have gone `2.4.9 -> 3.0.0`).

**Additional context**
Just sad to have to choose between updating with a mod I love or sticking with a colony in its prime that I've sunk a fair chunk of time into is all. I appreciate your hard work either way -- I know OSS maintainers get a lot more hate and a lot less love than they deserve, so I hope I don't come across as piling that on.

**Comment by zymex22** [comment]
its noted and we have talked about how to be more clear about it in future.
that said this should be only game breaking change we should need to make.

but everything on this one went sideways after it became such a big update, it was never planned to be that big and so many changes but after thornsworth passed away it got delayed and we just kept working on it and make it better and try to take a important step for a more "stable release"

as stated on steam page and github its a unstable build / WIP.
that may break saves, but we try to avoid it.

biggest issue that break saves with this update is removal of coal that appently goes deep into Vanilla RW and can corrupt things.
i just after remembered ppl can just add WD simple coal to solve this.

but items will still be deleted though but thats a one time error/delete.


- anyways we will be more clear next time and try use that Hugslib popup.
- Closed
